homeassistant/components/baidu/manifest.json

```diff
@@ -1,6 +1,6 @@
 {
   "domain": "baidu",
-  "name": "Baidu",
+  "name": "Baidu TTS",
```

Comment:
Shouldn't this just be Baidu, since it could also provide other things in the future ?

Reply:
Sure!